packages/terra-profile-image/tests/jest/__snapshots__/ProfileImage.test.jsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`should render a Terra image with a placeholder prop 1`] = `
 <div>
   <Image
     alt="A profile image"
+    fit="fill"
```

Comment:
@pranav300 I'd expect this to always be `fit="cover"` like avatar (profile image was an older component that was used before avatar but in the same way), so lets add to it to `/ProfileImage.jsx` and update snapshot again.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/18dd5b93d46d856d085bf005ebcac7b4ac2b3685